src/webrtc.ts

```diff
@@ -437,7 +437,7 @@ export class WebrtcExtension extends EventEmitter implements StreamingClientExte
       });
       this.currentMaxStatSize = desiredMaxStatsSize;
     } catch (err: any) {
-      if (err.status === 413) {
+      if (err.response.status === 413) {
```

Comment:
Should I mimic what was done in client logger? Cover both `err.status` and `err.response.status` along with the number and string?

Reply:
No, I think we're good here. This is already a number and isn't being changed to a string so I don't think we need to check strings at all.